imod/tests/test_mf6/test_mf6_drn.py

```diff
@@ -465,3 +470,28 @@ def test_html_repr(drainage):
     html_string = imod.mf6.Drainage(**drainage)._repr_html_()
     assert isinstance(html_string, str)
     assert html_string.split("</div>")[0] == "<div>Drainage"
+
+
+@pytest.mark.usefixtures("imod5_dataset")
+def test_from_imod5(imod5_dataset, tmp_path):
+    target_dis = StructuredDiscretization.from_imod5_data(imod5_dataset)
+    target_npf = NodePropertyFlow.from_imod5_data(
+        imod5_dataset, target_dis.dataset["idomain"]
+    )
+
+    drn_2 = imod.mf6.Drainage.from_imod5_data(
+        "drn-2",
+        imod5_dataset,
+        target_dis,
+        target_npf,
+        allocation_option=ALLOCATION_OPTION.at_elevation,
+        distributing_option=DISTRIBUTING_OPTION.by_crosscut_thickness,
+        regridder_types={},
+    )
+
+    # check the number of produced pacages is correct
+    assert drn_2 is not None
```

Comment:
Comment doesn't coincide with what is tested, I think a leftover comment from the previous review iteration? A more specific test here is:

```suggestion
    assert isinstance(drn_2, imod.mf6.Drainage)
```